docs/es/week12/12-3.md

```diff
@@ -0,0 +1,546 @@
+
+<!-- The components of the vector $\vect{a}$ are also called "scores" because the scalar product between two vectors tells us how aligned or similar two vectors are. Therefore, the elements of $\vect{a}$ provide information about the similarity of the overall set to a particular $\boldsymbol{x}_i$. -->
+Los componentes del vector $\vect{a}$ son también llamados "puntajes" porque el producto escalar entre dos vectores nos dice que tan similares parecen ser. Por lo tanto, los elementos de $\vect{a}$ proveen información sobre la similitud del conjunto en su totatalidad con respecto a un $\boldsymbol{x}_i$ en particular.
+
+<!-- The square brackets represent an optional argument. Note that if $\arg\max(\cdot)$ is used, we get a one-hot vector of alphas, resulting in hard attention. On the other hand, $\text{soft(arg)max}(\cdot)$ leads to soft attention. In each case, the components of the resulting vector $\vect{a}$ sum to 1. -->
+Los corchetes representan un argumento opcional. Nótese que si se utiliza $\arg\max(\cdot)$, obtenemos un vector con codificación "one-hot" de alfas, lo cual resulta en atención intensa. Por otro lado, $\text{soft(arg)max}(\cdot)$ resulta en atención ténue. En ambos casos, la suma de los componentes del vector resultante $\vect{a}$ es igual a 1.
+
+<!-- Generating $\vect{a}$ this way gives a set of them, one for each $\boldsymbol{x}_i$. Moreover, each $\vect{a}_i \in \mathbb{R}^t$ so we can stack the alphas in a matrix $\boldsymbol{A}\in \mathbb{R}^{t \times t}$. -->
+Al generar $\vect{a}$ de esta manera, se produce un conjunto de los mismos, uno por cada $\boldsymbol{x}_i$. Además, cada $\vect{a}_i \in \mathbb{R}^t$ de modo que podemos apilar las alfas en la matriz $\boldsymbol{A}\in \mathbb{R}^{t \times t}$.
+
+<!-- Since each hidden state is a linear combination of the inputs $\boldsymbol{X}$ and a vector $\vect{a}$, we obtain a set of $t$ hidden states, which we can stack into a matrix $\boldsymbol{H}\in \mathbb{R}^{n \times t}$. -->
+Puesto que cada estado oculto es una combinación lineal de las entradas $\boldsymbol{X}$ y un vector $\vect{a}$, obtenemos un conjunto $t$ de estados ocultos, los cuales podemos apilar en una matriz $\boldsymbol{H}\in \mathbb{R}^{n \times t}$.
+
+$$
+\boldsymbol{H}=\boldsymbol{XA}
+$$
+
+
+<!-- ## [Key-value store](https://www.youtube.com/watch?v=f01J0Dri-6k&t=1056s) -->
+## [Almacén de clave/valor](https://www.youtube.com/watch?v=f01J0Dri-6k&t=1056s)
```

Comment:
I'm not convinced about the translation for "key-value store" that I used here. Suggestions?

Reply:
I think you may want to discuss on Slack. I'm not a native speaker.

Reply:
I've never heard anyone saying it in Spanish, but "Almacén de clave/valor" or "Base de datos clave/valor" seems to be right.
For reference:
[https://aws.amazon.com/es/nosql/key-value/](https://aws.amazon.com/es/nosql/key-value/)